packages/svg/src/icons/types.ts

```diff
@@ -0,0 +1,15 @@
+type Properties<K extends string | number, T> = {
+    [P in K]: T;
+};
+
+type CSSProperties = Properties<string | number, string & {}>
+
+export type IconProps = {
+    width?: string | number;
+    height?: string | number;
+    color?: string;
+    fill?: string;
+    style?: CSSProperties | undefined;
+    onClick?: () => void;
+    className?: string;
+}
```

Comment:
The Icon Types should inherit from svg ( Our  should be actually just wrappers over svgs )  

```suggestion
import { SVGAttributes } from 'react';
export interface IconProps extends SVGAttributes<SVGElement> {
    children?: never;
    color?: string;
    title?: string;
}
```